scene/gui/line_edit.cpp

```diff
@@ -2990,6 +3001,10 @@ LineEdit::LineEdit(const String &p_placeholder) {
 	set_placeholder(p_placeholder);
 
 	set_editable(true); // Initialize to opposite first, so we get past the early-out in set_editable.
+
+#ifdef TOOLS_ENABLED
+	set_editing_on_text_submit(!(Engine::get_singleton()->is_editor_hint() || Engine::get_singleton()->is_project_manager_hint()));
```

Comment:
Found that `main.cpp` falls back to project manager when running the `--doctool`.